docs.mk

```diff
@@ -0,0 +1,70 @@
+pages   := $(shell find . -type f -name '*.adoc')
```

Comment:
I suggest to move this Makefile do the documentation folder, just like the e2e Makefile is in the `/e2e` folder

Reply:
Thanks for the hint.